scripts/partition_manager/partition_manager.rst

```diff
@@ -4,7 +4,7 @@ Partition Manager
 #################
 
 Partition Manager is a Python script that sets the start address and size of all image partitions in a multi-image build context.
-When creating an application that requires child images (for example, a bootloader), you can configure Partition Manager to control where in memory each image should be placed.
+When creating an application that requires child images (for example, a bootloader), you can configure Partition Manager to control where the images are placed in flash memory.
```

Comment:
Should we drop `memory`, and simply write `the images are placed in flash.` ?

Reply:
No, "flash memory" is more precise and easier to understand.